### ISSUE TITLE
Deleting a job should perhaps decrement the count for the current period

### ISSUE THREAD

**Issue by mihaijoldis** [restrictcontentpro/rcp-wp-job-manager#27]
Mentioned by a user and i was able to test this. Say you add a limit of 3 jobs for the membership.
Customer purchases today and posts 3 jobs. He's now at this limit and can't post any more.

However he decides to remove one job few hours later so he can post another one but he can't as the counter still counts 3 posted jobs.

While writing this it just came to my mind that if we make it like that it might be "exploitable" as in user might be able to fill in the position in a few days then delete the job and post another one.

**Comment by nosegraze** [comment]
> While writing this it just came to my mind that if we make it like that it might be "exploitable" as in user might be able to fill in the position in a few days then delete the job and post another one.

Yes that was my concern as well.

1. The time between job creation and deletion is extremely relevant. A few minutes apart and this makes sense. But a few days apart and this is definitely exploitable.
2. Another time-related thing to be careful of is that maybe you submit a job January 30th, renewal comes in January 31st and resets your count, then you delete a job on February 1st... You're now in a new billing period. Do we still decrement?

I'm more inclined to not decrement at all. I'd rather make the current count editable by admins so they can manually adjust it if they want. But doing it automatically has too many variables for me to be really comfortable doing it.

**Comment by mihaijoldis** [comment]
Agreed. Something perhaps that can be adjusted/membership level individually ?

**Comment by nosegraze** [comment]
Perhaps some kind of grace period setting.
